src/python_src/util/ml_classifier.py

```diff
@@ -2,26 +2,58 @@
 import os
 import re
 import string
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, List
 
+import boto3
 import joblib
 import onnxruntime as ort
 from numpy import float32, ndarray
 
+from src.python_src.util import app_utilities
+
 
 class MLClassifier:
-    def __init__(self, model_file: str, vectorizer_file: str):
+    def __init__(self, model_file: str = "", vectorizer_file: str = "", model_directory_path: str = ""):
+        model_file, vectorizer_file, model_directory_path = self.download_models_from_s3(
+            model_file, vectorizer_file, model_directory_path
+        )
+
         if not os.path.exists(model_file):
             raise Exception(f"File not found: {model_file}")
         if not os.path.exists(vectorizer_file):
             raise Exception(f"File not found: {vectorizer_file}")
         self.session = ort.InferenceSession(model_file)
         self.vectorizer = joblib.load(vectorizer_file)
 
-    def make_predictions(self, conditions: list[str]) -> List[Tuple[str, float]] | Any:
-        """Returns a list of the predicted classification names and the probability, for example:
-        [('Musculoskeletal - Wrist', 'Eye (Vision)', .971), ('Hearing Loss', .741)]
+    def download_models_from_s3(
+        self, model_file: str = "", vectorizer_file: str = "", model_directory_path: str = ""
+    ) -> tuple[str, str, str]:
+        if not model_directory_path:
+            model_directory_path = app_utilities.app_config["ml_classifier"]["data"]["directory"]
+        os.makedirs(model_directory_path, exist_ok=True)
+        if not model_file:
+            model_file = app_utilities.app_config["ml_classifier"]["model_file"]
+        if not vectorizer_file:
+            vectorizer_file = app_utilities.app_config["ml_classifier"]["vectorizer_file"]
+        try:
+            s3_client = boto3.client("s3")
+            s3_client.download_file(
+                app_utilities.app_config["ml_classifier"]["aws"]["bucket"],
+                app_utilities.app_config["ml_classifier"]["aws"]["model"],
+                model_file,
+            )
+            s3_client.download_file(
+                app_utilities.app_config["ml_classifier"]["aws"]["bucket"],
+                app_utilities.app_config["ml_classifier"]["aws"]["model"],
```

Comment:
```suggestion
                app_utilities.app_config["ml_classifier"]["aws"]["vectorizer"],
```
should this be the `vectorizer` key?
if so, can we add a unit test for this?